RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -197,17 +197,20 @@
 fastSim.toModify(jetCoreRegionalStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
 
 # Final sequence
-JetCoreRegionalStep = cms.Sequence(cms.ignore(jetsForCoreTracking)*
-                                   cms.ignore(firstStepGoodPrimaryVertices)*
-                                   #jetCoreRegionalStepClusters*
-                                   jetCoreRegionalStepSeedLayers*
-                                   jetCoreRegionalStepTrackingRegions*
-                                   jetCoreRegionalStepHitDoublets*
-                                   jetCoreRegionalStepSeeds*
-                                   jetCoreRegionalStepTrackCandidates*
-                                   jetCoreRegionalStepTracks*
-#                                   jetCoreRegionalStepClassifier1*jetCoreRegionalStepClassifier2*
+JetCoreRegionalStepTask = cms.Task(#cms.ignore(jetsForCoreTracking)*
```

Comment:
is there any value from ``` #cms.ignore ``` ?
It looks like it can be removed

Reply:
not sure with task, with sequence ```jetsForCoreTracking'''  acted as filter and triggered reconstruction even if final tracking product were never used...

Reply:
Technically in sequence `cms.ignore(jetsForCoreTracking)` acts as a producer. With tasks IIRC the "filter decision" of EDFilters is ignored "by definition" (i.e. what `cms.ignore` does).

To me the commented line has no value.

Reply:
As I recall that is what Chris told me. This is why I commented out the cms.ignore() and added the EDFilter directly.

Reply:
I agree with Matti, I do not see a value in the comments which have `#cms.ignore(...)` where the next line is just the same module label as in the call.